snapcraft/commands/microcloud

```diff
@@ -7,4 +7,7 @@ if [ -x "${SNAP_COMMON}/microcloud.debug" ]; then
     mkdir -p "${GOCOVERDIR}"
 fi
 
+# Use nano editor for interactive edits.
+export EDITOR="nano -f /snap/microcloud/current/etc/nanorc"
```

Comment:
Would you mind throwing `--restricted` in there as well? See https://man.archlinux.org/man/nano.1 for what it does.
This was used by LXD before we completely removed the bundled `nano`.